### ISSUE TITLE
Request for training and benchmark code

### ISSUE THREAD

**Issue by Phil-Fan** [HaxxorCialtion/SimpleTool#2]
Hi authors,

Thanks for open-sourcing SimpleTool. Would it be possible to release the training code and benchmark scripts used in the paper? They would be very helpful for reproducing the reported accuracy and latency results.

An estimated release timeline would also be appreciated. Thank you!
